proto/allocation/allocation.proto

```diff
@@ -15,16 +15,28 @@
 syntax = "proto3";
 
 package allocation;
+option go_package = "./allocation";
 
 import "google/api/annotations.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
+  info: {
+    title: "proto/allocation/allocation.proto";
+    version: "version not set";
+  };
+  schemes: HTTPS;
+  consumes: "application/json";
+  produces: "application/json";
+};
```

Comment:
`grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger` definition is used in swagger.json definition